corehq/apps/export/static/export/js/list_exports.ng.js

```diff
@@ -80,6 +80,11 @@
             var currentUrl = useLegacyBulkExportUrl ? $scope.legacy_bulk_download_url : $scope.bulk_download_url;
             input.closest("form").attr("action", currentUrl);
         };
+        $scope.downloadRequested = function($event) {
+            var $btn = $($event.target);
+            $btn.addClass('disabled');
```

Comment:
i don't think you need to switch the href in this case. the `disabled` class adds `pointer-events: none;` which means they can't click it